src/components/molecules/Common/Notification/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+
```

Comment:
Should this component be a molecule of our system?
I assume this component doesn't have any knowledge about Re:Earth and should be an atom.

Reply:
It is using multiple atom components which we don't explicitly prohibit (I don't think), but I feel like we implicitly prohibit it since atoms should be the "smallest", most basic piece. Whereas molecules are built up of atoms and other molecules.
So I will argue that it is exactly where it should be ;)

Reply:
Based on the typical atomic design principle, I agree that this is a molecule. But our component classification rules, atoms can depend on atoms and the difference between atoms and molecules is if they know the existence of Re:Earth.  I assume this component is more general and can be used in other apps as well so I thought it's an atom.
@rot1024 How do you think?

Reply:
Turns out it is an Atom